samples/led-blink/led-blink.csproj

```diff
@@ -2,12 +2,26 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RootNamespace>led_blink</RootNamespace>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
+  <!--  This sample uses conditional ItemGroups, but you only need one of them.
+        The ProjectReference is used by the repo build, to ensure that the samples work
+        with new builds of the System.Device.Gpio project.
+        The PackageReference is intended for most user use-cases.
+        If you want to test the package, type `dotnet build /p:PreferPackageReference=true`
+        If you want to copy this project and use outside of the repo, then:
+
+          1. Delete the first ItemGroup.
+          2. Remove the Condition from second ItemGroup
+   -->
+
+  <ItemGroup Condition="!$(PreferPackageReference)">
     <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(PreferPackageReference)">
+    <PackageReference Include="System.Device.Gpio" Version="1.1.0-prerelease.20153.1" />
+  </ItemGroup>
+
```

Comment:
Maybe something like this, to simplify the "if you're copy-pasting this for your own example" steps?

```suggestion
  <ItemGroup>
    <PackageReference Include="System.Device.Gpio" Version="1.1.0-prerelease.20153.1" />
  </ItemGroup>

  <!--  This sample uses conditional references, but you only need one of them.
        The ProjectReference is used by the repo build, to ensure that the samples work
        with new builds of the System.Device.Gpio project.
        The PackageReference is intended for most user use-cases.
        If you want to test the package, type `dotnet build /p:PreferPackageReference=true`
        If you want to copy this project and use outside of the repo, then
        delete the second ItemGroup.
   -->
  <ItemGroup Condition="'$(PreferPackageReference)' != 'true'">
    <PackageReference Remove="System.Device.Gpio" />
    <ProjectReference Include="$(MainLibraryPath)System.Device.Gpio.csproj" />
  </ItemGroup>


```